modules/libcom/src/iocsh/iocsh.h

```diff
@@ -49,22 +49,47 @@ extern "C" {
 #endif
 
 /**
- * @enum iocshArgType
- *
  * This typedef lists the values that can be used as argument data types
- * when building the piocshFuncDef parameter of iocshRegister().
- * 
- * @code {.cpp}
- * static const iocshArg AsynGenericConfigArg0 = {"Port Name", iocshArgString};
- * static const iocshArg AsynGenericConfigArg1 = {"Number Devices", iocshArgInt};
- * @endcode
+ * when building the first parameter of iocshRegister().
+ *
+ * ```
+ * static const iocshArg AsynGenericConfigArg0 = {
+ *     // name
+ *     "Port Name",
+ *     // type
+ *     iocshArgString,
+ * };
```

Comment:
You could save some vertical space here with something like this:
```C
 * static const iocshArg AsynGenericConfigArg0 = {
 *     // name, type
 *     "Port Name", iocshArgString,
 * };
```
although to be honest I don't see that comment line adding much clarity. I do prefer that the struct member data appears on a separate line though, the original is harder to read.